tests/tt_metal/tt_metal/dispatch/dispatch_program/test_ProgramReuse.cpp

```diff
@@ -0,0 +1,368 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <algorithm>
+#include <cmath>
+#include <functional>
+#include <random>
+
+#include "command_queue_fixture.hpp"
+
+using std::vector;
+using namespace tt;
```

Comment:
1. Let's not shorten `std::`, otherwise the usage is very inconsistent in the codebase (in this test file also).
2. Can't we write the test in `namespace tt::tt_metal {...}` directly?

Reply:
Updated